sdv/data_processing/data_processor.py

```diff
@@ -97,9 +97,10 @@ def to_dict(self):
             dict:
                 Dict representation of this DataProcessor.
         """
+        constraints_to_reverse = [cnt.to_dict() for cnt in self._constraints_to_reverse]
```

Comment:
minor: does it not fit to do `constraint` instead of `cnt`. In general I think we should try to use full descriptive names